### ISSUE TITLE
Autocomplete: internal select doesn't have a label

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2223]
That select isn't shown to end users, but not having a label is a problem per WCAG2 1.3.1 (A), reported as E885 by TotalValidator. Maybe adding an empty `title` is enough.

**Comment by avernet** [comment]
- TotalValidator says a `role="presentation"` is invalid on a `<select>`.
- Putting a `role="presentation"` on an ancestor has no effect.
- Adding `style="display: none"` on the `<select>` has no effect.
- Adding `title=""` as no effect.
- But: adding `title="internal"` works.
